packages/lib/responseQueue.ts

```diff
@@ -80,6 +81,9 @@ export class ResponseQueue {
           },
           this.config.apiHost
         );
+        if (responseUpdate.displayId) {
```

Comment:
We already have the displayId in the surveyState and don't need to get it through the responseUpdate. Please use this :-)